ds-caselaw-ingester/lambda_function.py

```diff
@@ -477,10 +511,8 @@ def handler(event, context):
 
     tar = tarfile.open(filename, mode="r")
     metadata = extract_metadata(tar, consignment_reference)
-
-    if not message.is_v1():
-        # this is just for debug purposes, it should be safely removable
-        store_file(open(filename, mode="rb"), "v2-debug", "debug.tar.gz", s3_client)
+    message.update_consignment_reference(metadata["parameters"]["TRE"]["reference"])
```

Comment:
Can we just adjust `get_consignment_reference()` for s3message rather than create this new method `update_consignment_reference`?

Reply:
There might be other approaches here, might be worth a chat.